### ISSUE TITLE
`npx typewriter init` does nothing

### ISSUE THREAD

**Issue by alete89** [segmentio/typewriter#249]
running `npx typewriter init` in an empty project directory with a package.json and `analytics-node` installed does nothing else than showing a message.

running on Ubuntu 20.04, Node 16.13.2, npm 8.3.1, analytics-node 6.0.0, typewriter 7.4.1.

![typewriter](https://user-images.githubusercontent.com/4999277/171044911-eecb5eb9-153e-4eaf-88c7-a31945ee1db7.gif)

**Comment by giovanna-guimaraes** [comment]
So, I'm having the exact same problem (running on Ubuntu 20.04, Node 16.15.1, npm 8.11.0) and I've so far tried with multiple terminal emulators without any success, but for some some reason the command works perfectly everywhere when I run it with sudo :clown_face: 

Anyway, as my friend @adrifmonte pointed out, the interactive prompt is there, it's just not visible. So at each step I copied the content and pasted it into a text editor, just so I could see what I was doing. It was a pain and I know it's not a real solution, but at least I was able to complete every step :shrug:

**Comment by oscb** [comment]
@giovanna-guimaraes agree, not an ideal solution. Does this work in the new v8 release for you? I'll give it a try on Ubuntu, we switched out of the old rendering system (which was kind of an overkill for typewriter to be sincere).

**Comment by douglasliralima** [comment]
What I do to solve this is use `FORCE_COLOR=0 npx typewriter init` to force it to be visible
